sell-loot.lic

```diff
@@ -60,30 +81,26 @@ class SellLoot
     walk_to @hometown['exchange']['id']
     release_invisibility
     exchange_to = @hometown['currency']
-    %w[dokoras kronars lirums]
-      .reject { |currency| currency == exchange_to }
+    $CURRENCIES
+      .reject { |currency| currency =~ /#{exchange_to}/i }
       .each { |currency| fput "exchange all #{currency} for #{exchange_to}" }
   end
 
   def deposit_coins(keep_copper)
     walk_to @hometown['deposit']['id']
     release_invisibility
     bput('wealth', 'Wealth:')
-    case bput('deposit all', 'you drop all your', 'You hand the clerk some coins', "You don't have any", 'There is no teller here', 'reached the maximum balance I can permit')
+    case bput('deposit all', 'you drop all your', 'You hand the clerk some coins', "You don't have any", 'There is no teller here')
```

Comment:
Why was this line changed?

Reply:
Whoops, I had a rebase mishap so used my old code on updated master and didn't notice this change.

I'll update.